.github/workflows/auto-release.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           fetch-depth: '0'
 
       - name: Bump version
-        uses: anothrNick/github-tag-action@1.67.0
+        uses: billyb2/github-tag-action
```

Comment:
hey, if we are going to use a fork of github-tag-action. Can it be under superfly org so everyone in Fly is able to maintain it?

Reply:
yep, sounds good!